libs/usb/LibUsbAdaptor.cpp

```diff
@@ -80,10 +80,22 @@ bool OpenHandleAndClaimInterface(libusb_device *usb_device,
   }
 
   int r = libusb_claim_interface(*usb_handle, interface);
-  if (r) {
+  if (r == LIBUSB_ERROR_BUSY) {
+	int error = libusb_detach_kernel_driver(*usb_handle, interface);
+	if (error) {
+		OLA_WARN << "Failed to detach kernel driver " << interface
+				 << " on device: " << usb_device;
+	} else {
+	  r = libusb_claim_interface(*usb_handle, interface);
+	}
+  }
+
+  if (r)
+  {
```

Comment:
Can you move this to the line above please.

Reply:
The other tab/space lint errors will also need fixing before merge:
https://travis-ci.org/OpenLightingProject/ola/jobs/404585543#L2174